dotCMS/src/main/webapp/WEB-INF/messages/Language_es.properties

```diff
@@ -1888,6 +1888,9 @@ logout-as = Salir de
 Logout = Desloguear
 Lucene-Query = Query de Lucene
 mail-domain = Dominio de correo
+email-address-validate = Enviar correo de prueba
```

Comment:
We shouldn't modify Language.properties for specific languages.

Reply:
why not? I always modify Eglish and Spanish